### ISSUE TITLE
no answer if the Russian alphabet is used

### ISSUE THREAD

**Issue by vchuchkalov** [bbyars/mountebank#477]
It's bug only for version mountebank >=2.1.0
On version <=2.0.0 it's not reproduce

#### Expected behaviour
return response

#### Actual behaviour
no response

#### Steps to reproduce
0. install mb in your package.json
1. create imposter file with cirillic symbols(russian language)
example: 
```
{
  "port": 3333,
  "protocol": "http",
  "stubs": [
    {
      "predicates": [{
        "deepEquals": {
          "body": { "тест": "2" }
        }
      }],
      "responses": [{
        "is": {
          "body": "Entire array matched"
        }
      }]
    }
  ]
}

```
2. run your mb with command ```npx mb --configfile='./imposter.json'```
3. send request to http://localhost:3333/ with body { "тест": "2" }

#### Software versions used

```
OS         : macOS
mountebank : 2.1.2
node.js    : 10.16.3
Installation method : npm
```

#### Log contents in mb.log when running mb --loglevel debug
```
{"message":"[mb:2525] mountebank v2.1.2 now taking orders - point your browser to http://localhost:2525/ for help","level":"info","timestamp":"2019-11-06T12:48:36.907Z"}
{"message":"[mb:2525] config: {\"options\":{\"configfile\":\"./imposter.json\",\"loglevel\":\"debug\",\"port\":2525,\"noParse\":false,\"pidfile\":\"mb.pid\",\"nologfile\":false,\"logfile\":\"mb.log\",\"allowInjection\":false,\"localOnly\":false,\"ipWhitelist\":[\"*\"],\"mock\":false,\"debug\":false,\"heroku\":false,\"protofile\":\"protocols.json\"},\"process\":{\"nodeVersion\":\"v10.16.3\",\"architecture\":\"x64\",\"platform\":\"darwin\"}}","level":"debug","timestamp":"2019-11-06T12:48:36.908Z"}
{"message":"[mb:2525] PUT /imposters","level":"info","timestamp":"2019-11-06T12:48:36.925Z"}
{"message":"[mb:2525] ::ffff:127.0.0.1:49527 => {\"imposters\":[{\"port\":3333,\"protocol\":\"http\",\"stubs\":[{\"predicates\":[{\"deepEquals\":{\"body\":{\"тест\":\"2\"}}}],\"responses\":[{\"is\":{\"body\":\"Entire array matched\"}}]}]}]}","level":"debug","timestamp":"2019-11-06T12:48:36.946Z"}
{"message":"[http:3333] Open for business...","level":"info","timestamp":"2019-11-06T12:48:36.954Z"}
{"message":"[http:3333] ::1:49528 ESTABLISHED","level":"debug","timestamp":"2019-11-06T12:48:42.989Z"}
{"message":"[http:3333] ::1:49529 ESTABLISHED","level":"debug","timestamp":"2019-11-06T12:48:42.989Z"}
{"message":"[http:3333] ::1:49528 LAST-ACK","level":"debug","timestamp":"2019-11-06T12:48:42.990Z"}
{"message":"[http:3333] ::1:49529 => POST /","level":"info","timestamp":"2019-11-06T12:48:42.990Z"}
{"message":"[http:3333] ::1:49529 => {\"requestFrom\":\"::1:49529\",\"method\":\"POST\",\"path\":\"/\",\"query\":{},\"headers\":{\"Content-Type\":\"text/plain\",\"User-Agent\":\"PostmanRuntime/7.18.0\",\"Accept\":\"*/*\",\"Host\":\"localhost:3333\",\"Accept-Encoding\":\"gzip, deflate\",\"Content-Length\":\"19\",\"Connection\":\"keep-alive\"},\"body\":\"{ \\\"ÑÐµÑÑ\\\": \\\"2\\\" }\",\"ip\":\"::1\"}","level":"debug","timestamp":"2019-11-06T12:48:42.994Z"}
{"message":"[http:3333] no predicate match","level":"debug","timestamp":"2019-11-06T12:48:42.995Z"}
{"message":"[http:3333] generating response from {\"is\":{}}","level":"debug","timestamp":"2019-11-06T12:48:42.995Z"}
{"message":"[http:3333] ::1:49529 <= {\"statusCode\":200,\"headers\":{\"Connection\":\"close\"},\"body\":\"\",\"_mode\":\"text\"}","level":"debug","timestamp":"2019-11-06T12:48:42.996Z"}
{"message":"[http:3333] ::1:49528 CLOSED","level":"debug","timestamp":"2019-11-06T12:48:42.997Z"}
{"message":"[http:3333] ::1:49529 CLOSED","level":"debug","timestamp":"2019-11-06T12:48:42.997Z"}
```

**Comment by bbyars** [comment]
fixed as of 7059d4bf

**Comment by bbyars** [comment]
Thanks for the great bug report; it helped identify the offending code in a
very short amount of time.
Should be fixed now. You can test a pre-release version with

> npm install -g mountebank-dev@2.1.2-beta.3761

Other install options are available at the test site:
http://mountebank-dev.herokuapp.com/docs/install
